src/aluno/CadastroAlunoPage.tsx

```
@@ -0,0 +1,312 @@
+        console.log('Valores do formulário:', values)
+        AlunoService.save(values)
+            .then(() => {
+                toast.success('Aluno cadastrado com sucesso!')
+                void navigate('/')
+            })
+            .catch((error: AxiosError<ErrorResponse>) => {
+                toast.error('Erro ao cadastrar aluno: ' + error.message)
+            })
+    }
+
+    function onError(errors: any) {
```

ESLint

[error] Delete `··`

(prettier/prettier)